### PR TITLE
Registered formatter for proto3

### PR DESCRIPTION
**Pull request by d-enk**
#161

### CODE REVIEW

**Review by stefanvanburen — CHANGES_REQUESTED** [review]
hi @d-enk, thanks for the PR. This looks reasonable to me and works with the following change locally. I think after that it should be good to go?

(for others, it seems like zxh0/vscode-proto3 [registers a DocumentFormattingEditProvider for `proto3`](https://github.com/zxh0/vscode-proto3/blob/f808059028c870ffd0b38babdfafcd49752dee1e/src/proto3Main.ts#L90), we're just doing the same here).

File: src/extension.ts [hunk 1]
```diff
@@ -125,7 +125,9 @@ export function activate(context: vscode.ExtensionContext) {
     diagnosticCollection.set(document.uri, diagnostics);
   };
 
-  context.subscriptions.push(vscode.languages.registerDocumentFormattingEditProvider('proto', new Formatter(binaryPath)));
+  let formatter = new formatter_1.Formatter(binaryPath);
```
[the remark this answers is not in the document]
Comment (d-enk) [reply src/extension.ts]: Wow.. Accidentally made changes from my local extension. Sorry)

**Review by stefanvanburen — APPROVED** [review]

File: src/extension.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (stefanvanburen) [reply src/extension.ts]: no worries at all :-)

**Comment by stefanvanburen** [comment]
hi @d-enk, your fix should be in the v0.5.1 version of vscode-buf — thanks again!

**Pull request merged**